content/posts/weekly/September-2022.md

```diff
@@ -22,3 +22,8 @@ weight: 58
   packit now correctly handles pagination of API results. This resolves an issue that caused
   Fedora 37 not to be pointed to from any alias.
   ([packit#1704](https://github.com/packit/packit/pull/1704))
+
+## Week 37 (September 13th – September 19th)
+
+- Using the method `users_with_write_access` you can generate the set of users that have write access to the project and the method `has_write_access(user)` you can find out if the user has write access to the project. ([ogr#742](https://github.com/packit/ogr/pull/742))
```

Comment:
```suggestion
- With our library ogr you can now use the method `users_with_write_access` to generate a set of users that have write access to a project. Method `has_write_access(user)` can find out if a user has write access to the project. ([ogr#742](https://github.com/packit/ogr/pull/742))
```
Was really confused until I realized it's an ogr change. Let's make this obvious from the start.